source/frontend/src/features/properties/map/project/add/ProductSubForm.tsx

```diff
@@ -0,0 +1,75 @@
+import { FastDatePicker, Input, TextArea } from 'components/common/form';
+import { SectionField } from 'features/mapSideBar/tabs/SectionField';
+import { FormikProps } from 'formik';
+import React from 'react';
+import { Col, Row } from 'react-bootstrap';
+import styled from 'styled-components';
+import { withNameSpace } from 'utils/formUtils';
+
+import { ProjectForm } from './models';
+
+export interface IProductSubFormProps {
+  index: number;
+  nameSpace: string;
+  formikProps: FormikProps<ProjectForm>;
+}
+
+export const ProductSubForm: React.FunctionComponent<IProductSubFormProps> = ({
+  index,
+  nameSpace,
+  formikProps,
+}) => {
+  const costEstimate = formikProps.values.products[index].costEstimate;
+  return (
+    <>
+      <Row>
+        <Col>
+          <SectionField label="Product code" required>
+            <Input field={withNameSpace(nameSpace, 'code')} />
+          </SectionField>
+        </Col>
+        <Col>
+          <SectionField label="Name" labelWidth="2" required>
+            <Input field={withNameSpace(nameSpace, 'description')} />
+          </SectionField>
+        </Col>
+      </Row>
+      <SectionField label="Start date" labelWidth="2" contentWidth="4">
+        <FastDatePicker field={withNameSpace(nameSpace, 'startDate')} formikProps={formikProps} />
+      </SectionField>
+      <Row>
+        <Col>
+          <SectionField label="Cost estimate">
+            <Input field={withNameSpace(nameSpace, 'costEstimate')} />
+          </SectionField>
+        </Col>
+        <Col>
+          {costEstimate?.toString() !== '' && (
+            <SectionField label="Estimate date" required>
+              <FastDatePicker
+                field={withNameSpace(nameSpace, 'costEstimateDate')}
+                formikProps={formikProps}
+              />
+            </SectionField>
+          )}
+        </Col>
+      </Row>
+      <SectionField label="Objectives" labelWidth="12">
+        <MediumTextArea field={withNameSpace(nameSpace, 'objective')} />
+      </SectionField>
+      <SectionField label="Scope" labelWidth="12">
+        <MediumTextArea field={withNameSpace(nameSpace, 'scope')} />
+      </SectionField>
+    </>
+  );
+};
+
+export default ProductSubForm;
+
+export const MediumTextArea = styled(TextArea)`
```

Comment:
maybe a candidate for a shared style?

Reply:
Yes, not sure if the new mockups are coming with this or is it just a thing for these forms but it could be moved into the general forms section